hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -664,6 +668,40 @@ def alter_status(table_name)
       puts 'Done.'
     end
 
+    #----------------------------------------------------------------------------------------------
+    # Use our internal logic to convert from "spec string" format to a coprocessor descriptor
+    #
+    # Provided for backwards shell compatibility
+    #
+    # @param [String] spec_str
+    # @return [ColumnDescriptor]
+    def coprocessor_descriptor_from_spec_str(spec_str)
```

Comment:
If you want to do this, please place a fat warning in the docs over on the `TableDescriptorBuilder.toCoprocessorDescriptor` method.

Reply:
Done